protos/sources/data-node/api/v2/trading_data.proto

```diff
@@ -1813,9 +1803,9 @@ message OrderEdge {
 // Request for listing margin levels
 message ListMarginLevelsRequest {
     // Party ID for which to list the margin levels
-    string party_id = 1;
+    string party_id = 1 [(google.api.field_behavior) = REQUIRED];
```

Comment:
Again I can list all margin levels across all markets and parties:
```
https://api.n06.testnet.vega.xyz/api/v2/margin/levels
```
so these aren't required.

Reply:
Address https://github.com/vegaprotocol/vega/pull/8070/commits/16a86a18b2d74fe341c6e3d4e19eb7563618cc95